packages/core/src/App/Containers/RealAccountSignup/address-details.jsx

```diff
@@ -83,11 +83,21 @@ class AddressDetails extends React.Component {
                 validateOnMount
                 onSubmit={(values, actions) => {
                     if (values.address_state) {
-                        values.address_state = this.props.states_list.length
-                            ? this.state.address_state_to_display
-                                ? getLocation(this.props.states_list, this.state.address_state_to_display, 'value')
-                                : getLocation(this.props.states_list, values.address_state, 'value')
-                            : values.address_state;
+                        let add_state;
+                        if (this.props.states_list.length) {
```

Comment:
Any way to simplify this if else section (87 - 100) a bit? I think we can move the `this.props.states_list.length` up one step and get rid of the first ternary.